specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-04-01-preview/costmanagement.json

```diff
@@ -3154,6 +4124,32 @@
       "required": true,
       "type": "string",
       "x-ms-parameter-location": "method"
+    },
+    "scopeViewParameter": {
+      "name": "scope",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The scope associated with view operations. This includes '/subscriptions/{subscriptionId}' for subscription scope, '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}' for resourceGroup scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}' for Billing Account scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/departments/{departmentId}' for Department scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/enrollmentAccounts/{enrollmentAccountId}' for EnrollmentAccount scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}' for BillingProfile scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/invoiceSections/{invoiceSectionId}' for InvoiceSection scope and '/providers/Microsoft.Management/managementGroups/{managementGroupId}' for Management Group scope.",
```

Comment:
"The scope associated with view operations. Cost Management supports any scope above a subscription, from billing accounts and management groups down to subscriptions and resource groups. Learn more about scopes and how to obtain the resource ID for your scope at [Understanding and working with scopes](https://aka.ms/costmgmt/scopes)"

I realize Query, Dimensions, and possibly Exports don't use this today. I want to fix those, too. Bonus points if you can specify this once and have all of those other APIs use the same description. It's always the same across all APIs.